articles/healthcare-apis/includes/fhir-bulk-delete-operation.md

```diff
@@ -67,18 +68,35 @@ Sample response of successfully completed delete job.
             "part": [
                 {
                     "name": "Practitioner",
-                    "valueDecimal": 10.0
+                    "valueInteger64": 10.0
                 },
                 {
                     "name": "Specimen",
-                    "valueDecimal": 7.0
+                    "valueInteger64": 7.0
                 },
                 {
                     "name": "Device",
-                    "valueDecimal": 3.0
+                    "valueInteger64": 3.0
                 }
             ]
         }
     ]
 }
 ```
+## Bulk Delete operation error messages
+
+Here are the list of error messages that can occur if bulk delete operation fails
+
+|HTTP Status Code | Details | Recommended action |
+|-----------------|---------|--------------------|
+|202 |Bulk delete job is stuck | Cancel the bulk delete job and try again.|
+|500 |Connection to database failed | Create support ticket to resolve the issue.|
+|429 |Throttling errors | For Azure API for FHIR, you can increase RUs and retry the job. For Azure Health Data Services FHIR Server, you can try to delete a smaller amount of data at a time. |
+
+## FAQ
+1. Will API interactions see latency impact when bulk delete operation job is executed concurrently?
+
+When running a bulk delete operation there is the possibility to see increased latency on concurrent calls to the service. To avoid latency increase, the recommendation is to cancel the bulk delete job and rerun it during a period of lower traffic.
```

Comment:
In case of slower latency, does AHDS FHIR service scale to meet the demand?

Reply:
The load bulk delete puts on a service is at the database layer. The database can scale, but it works differently from our replica scaling. I think we need to wait for Mikael's perf tests to see how our database scaling reacts to bulk delete.